.pre-commit-config.yaml

```diff
@@ -89,6 +89,12 @@ repos:
           .*example_dags/.*|
         additional_dependencies: ['toml']
 
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.7.4
+    hooks:
+      - id: bandit
+        exclude: ^tests/.*\.py$|.*example_dags/.*
```

Comment:
Why did we exclude the `example_dags`?

Reply:
Temporarily added, as some of the example DAGs don't satisfy bandit but that is OK for now. 

I will create a separate GitHub issue to fix it